test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+set -a
+
+: ${RUBY_VERSION=2.2}
+sed "s/\${RUBY_VERSION}/$RUBY_VERSION/" Dockerfile > Dockerfile.$RUBY_VERSION
+docker-compose build --pull
+
+function finish {
+  docker-compose down
+}
+trap finish EXIT
+
+docker-compose pull pg possum 
+POSSUM_DATA_KEY="$(docker-compose run -T --no-deps possum data-key generate)"
+
+docker-compose up -d possum
+
+docker-compose run test ci/wait_for_server.sh
```

Comment:
I haven't seen `HEALTHCHECK` before. Yeah, it'll be good to add that, probably once something like `wait_for_server` becomes part of the possum container.